src/main/scala/com/fulcrumgenomics/umi/ReviewConsensusVariants.scala

```diff
@@ -160,6 +160,13 @@ class ReviewConsensusVariants
   private val refFile = ReferenceSequenceFileFactory.getReferenceSequenceFile(ref)
   private val dict = refFile.getSequenceDictionary
 
+  validate(this.refFile.isIndexed,
+    f"The reference file has not indexed (use `samtools faidx ref.fa`): ${this.ref}"
+  )
```

Comment:
Then the command can be copy-pasted and it will hopefully work.

```suggestion
  validate(this.refFile.isIndexed,
    f"The reference file has not indexed (use `samtools faidx ${this.ref}`)."
  )
```